### DISCUSSION TITLE
Unable to Reuse Assignments from Other Classrooms in Bulk

### DISCUSSION THREAD

**Discussion by khammami** [community/community#141141] [GitHub Education]
### Hi everyone,

After selecting the organization and the classroom where you want to reuse the selected assignments, it simply opens the target classroom page without any notifications or indications that the assignments are being copied. Consequently, the assignments are not copied even after waiting for 30 minutes. Last year, there was a message indicating that the assignments were being duplicated and to refresh the page later.

**Comment by jjswanky** [comment]
I am having the same issue. It only happens whenever an assignment has starter code. 

It appears to be a with how the starter code is copied from the template repo. When creating a new assignment starter code must be a template. But the copied repo created for the new assignment is not automatically made a template repository. Because the copied repo is not a template, classrooms fails to copy the assignment. The work around is go into the starter code repo after it is created for the new assignment and manually make it a template repository.

It would be nice if classrooms did this automatically.
